docs/DeveloperGuide.md

```diff
@@ -4,26 +4,162 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
-
+## Design
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Good Class
+#### Description
+The goods class keeps track of the various inventory that will be input into the system through the commands class.
```

Comment:
Yes as @khseah said, I also think denoting the class names as code would be good. In addition I think it would be better to capitalise it as well (e.g. `Good`)